junit-platform-launcher/src/main/java/org/junit/platform/launcher/listeners/MutableTestExecutionSummary.java

```diff
@@ -192,13 +194,13 @@ public void printFailuresTo(PrintWriter writer) {
 
 	@Override
 	public List<Failure> getFailures() {
-		return new ArrayList<>(failures);
+		return Collections.unmodifiableList(new ArrayList<>(failures));
```

Comment:
Since JUnit 5 does not have a 1.0 release yet, would it be alright to modify both the mutability and view/copy-ness of `getFailures()` as long as it's recorded in the release notes?